test/test_initial_design/test_initial_design.py

```diff
@@ -22,8 +22,12 @@ def setUp(self):
         self.cs.add_hyperparameter(UniformFloatHyperparameter(
             name="x1", lower=1, upper=10, default_value=1)
         )
-        self.scenario = Scenario({'cs': self.cs, 'run_obj': 'quality',
-                                  'output_dir': ''})
+        self.scenario = Scenario({
+            'cs': self.cs,
+            'run_obj': 'quality',
+            'output_dir': '',
+            'ta_run_limit': 100,
+        })
```

Comment:
Does this change anything?

Reply:
No, it is only necessary to test that the exception is triggered if we request an initial design larger than the number of target runs.